src/main/java/org/folio/rest/impl/util/OkapiResponseUtil.java

```diff
@@ -0,0 +1,75 @@
+package org.folio.rest.impl.util;
+
+import io.vertx.core.AsyncResult;
+
+import javax.ws.rs.core.Response;
+
+import org.folio.rest.jaxrs.model.Error;
+import org.folio.rest.jaxrs.model.Errors;
+
+/**
+ * Utility methods for Okapi Responses.
+ */
+public class OkapiResponseUtil {
+
+  private OkapiResponseUtil() {
+    throw new UnsupportedOperationException("Do not instantiate");
+  }
+
+  /**
+   * Parse an Okapi HTTP Response for error messages.
+   *
+   * This will handle only 4xx (client side) HTTP error messages.
+   *
+   * @param reply
+   *   The Okapi Response to parse for error messages.
+   *
+   * @return
+   *   A string of concatenated error messages associated with the reply.
+   *   A newline is appended at the end of each distinct error message.
+   */
+  public static String getErrorMessage(AsyncResult<Response> reply) {
+    String message = null;
```

Comment:
Why do you use null? You can simply return an empty string if no error message is found in Response.

Reply:
This is just me being cautious.

https://github.com/folio-org/mod-circulation-storage/blob/ac1e03e575b2407e167654ddd0ee69350da5db26/src/main/java/org/folio/rest/impl/util/RequestsApiUtil.java#L23

This code is considering a NULL instead of an empty string, and so I was worried that there may be places I haven't noticed that might be considering, expecting, or using NULL.

I would prefer to use an empty string, then I could just remove the assignment condition (`if (errors.getErrors().size() > 0)`).